File: documentation/user/en/query/requirements/examples/facet/facet-groups-conjunction.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					userFilter(
						facetHaving(
							"groups",
							entityHaving(
								attributeInSet("code", "sale")
							)
						)
					)
				),
				require(
					facetSummaryOfReference(
						"groups",
						IMPACT,
						entityFetch(
							attributeContent("code")
						)
					),
					facetGroupsConjunction("groups", WITH_DIFFERENT_FACETS_IN_GROUP)
				)
			)
		);
	}
);

File: documentation/user/en/query/requirements/examples/facet/facet-groups-conjunction.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		UserFilter(
        			FacetHaving(
        				"groups",
        				EntityHaving(
        					AttributeInSet("code", "sale")
        				)
        			)
        		)
        	),
        	Require(
        		FacetSummaryOfReference(
        			"groups",
        			Impact,
        			EntityFetch(
        				AttributeContent("code")
        			)
        		),
        		FacetGroupsConjunction("groups")
        	)
        )
	)
);